mutiny-core/src/lib.rs

```diff
@@ -220,16 +220,29 @@ impl<S: MutinyStorage> MutinyWallet<S> {
 
                 add_relay_res.expect("Failed to add relays");
                 client.connect().await;
-                client.subscribe(nostr.get_nwc_filters()).await;
+
+                let mut last_filters = nostr.get_nwc_filters();
```

Comment:
Just having this as mut and changing in place won't cause the filters to change for the subscription I don't think. Could just have it break and then it'll reconnect.

Reply:
What do you mean? The new filters are sent across. I'm working on the blastr side to make sure filters are handle more efficiently.

Reply:
Oh I missed that you were calling subscribe again. I guess that works. Maybe we want to cancel the previous subscription, guess it's not a big deal if we don't.